Modules/admin/admin_main_view.php

```diff
@@ -275,6 +275,9 @@ function row($title, $value, $title_css = '', $value_css='') {
         </dl>
 
         <h4 class="text-info text-uppercase border-top pt-2 mt-0 px-1"><?php echo _('Emoncms'); ?></h4>
+        <div>
+            <a href="<?php echo $path; ?>admin/settings" class="btn btn-info"><?php echo _('View Emoncms Settings'); ?></a>
+        </div>
```

Comment:
To address the comment of how to align right the code below will work (although it would be better to give the div a class and then add the style into admin_styles.css)
Change the first div tag too:
`<div style="float:right;">`
After the closing div tag add:
`<div class="clearfix"></div>`